src/connect.js

```diff
@@ -104,11 +112,28 @@ function connect(mapStoreStateToProps, mapDispatchToProps, mergeProps, options =
 			}
 
 			/**
+			 * Returns the wrapped dispatch props that should be passed to the
+			 * wrapped component.
+			 * @return {!object}
+			 * @protected
+			 */
+			getDispatchProps_() {
+				if (!this.wasRendered ||
```

Comment:
Maybe it wouldn't make a difference in the behavior, but I think it would be better to replace this with a check for `!this.dispatchProps_`, since that will be easier to understand when reading this code, than having to relate to the component having been rendered or not.